task_2/index.html

```diff
@@ -0,0 +1,502 @@
+              <img src="./images/TuskVentures-black.svg" alt="">
+            </div>
+            <blockquote>
+              “Skout helped us bring our security and operations forward dramatically by being so responsive and detail-oriented. The whole team is really great to work with.”
+              <p>
+                — Yoni Rechtman
+              </p>
+            </blockquote>
+          </div>
+          <div class="partner-slide">
+            <div class="partner-logo">
+              <img src="./images/TuskVentures-black.svg" alt="">
+            </div>
+            <blockquote>
+              “Skout helped us bring our security and operations forward dramatically by being so responsive and detail-oriented. The whole team is really great to work with.”
+              <p>
+                — Yoni Rechtman
+              </p>
+            </blockquote>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="feedback">
+      <div class="wrapp">
+        <div class="feedback-img"></div>
+        <div class="feedback-description">
+          <h3>
+            Still have questions? Reach out today.
+          </h3>
+          <p>
+            Whether you’re a potential management service provider or a business owner looking for a little extra peace of mind, our representatives are standing by to answer any questions and walk you through the benefits of partnering with us.
+          </p>
+        </div>
+        
+      </div>
+      <div class="feedback-interactivity">
+        <div class="">
+          <h5>
+            Ready to learn somthing moore?
+          </h5>
+          <div class="get-feedback">
+            <span>
+              Get in Touch
+            </span>
+            <button></button>
+          </div>
+        </div>
+        <div class="">
+          <h5>
+            Are you a MSP?
+          </h5>
+          <div class="get-feedback">
+            <span>
+              Become a Partner
+            </span>
+            <button></button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer class="footer">
+     <div class="wrapp">
+       <div class="footer-info">
+         <div class="footer-links">
+           <a href="#" class="footer-link">Blog</a>
+           <a href="#" class="footer-link">Company</a>
+           <a href="#" class="footer-link">Terms & Conditions</a>
+           <a href="#" class="footer-link">Privacy Policy</a>
+           <a href="#" class="footer-link">Careers</a>
+           <div class="call-number">
+             <span>CALL THE SOC</span>
+             <a href="tel:1-855-838-4500" class="number">1-855-838-4500</p>
+           </div>
+         </div>
+         <div class="company-adrress">
+           <div class="footer-links">
+             <div class="footer-info-title">New York</div>
+             <a href="+1 631-203-6600" class="number">+1 631-203-6600</a>
+             <address class="adrress">
```

Comment:
text should not be italic 
![Uploading image.png…]()

Reply:
Done